### ISSUE TITLE
The number of epochs 

### ISSUE THREAD

**Issue by aizest** [LaceyChen17/DREAM#8]
Hi, Lacey, 

It's great to find your neat implementation of the DREAM model.
By updating several lines of code, I'm able to run it on the Instacart dataset with PyTorch 0.4.1.  Just wonder how many epochs you have trained the model when you participated the Kaggle competition? And how much time did it take? In my case, the loss value on the test set starts at 0.66 and drops to 0.11 after 5 epochs, and it takes about 4.5 hours for each epoch. I'm using a Nvidia K80 GPU with 12 GB GPU memory. I just tried the "train_dream" function so far, not for "train_reorder_dream" yet.

The hyper-parameters I'm using are:

- 'basket_pool_type': 'max', 
- 'rnn_layers': 2,  
- 'rnn_type': 'LSTM', 
- 'dropout': 0.5,
- 'num_product': 49688 + 1 + 1, 
- 'none_idx': 49689,
- 'embedding_dim': 128, 
- 'cuda': True, 
- 'clip': 200,  
- 'epochs': 10,
- 'batch_size': 64, 
- 'learning_rate': 0.001, 
- 'log_interval': 100, 

Thanks a lot,
Xuan

**Comment by yihong-chen** [comment]
sorry for the late response. my run time was similar to you. BTW do you have any interest to upgrade it to pytorch 1.0? I guess it would be faster than the old versions.
